evalml/tests/model_understanding_tests/test_partial_dependence.py

```diff
@@ -517,7 +569,7 @@ def test_graph_partial_dependence_multiclass(
     for data, label in zip(fig_dict["data"], pipeline.classes_):
         assert len(data["x"]) == 5
         assert len(data["y"]) == 5
-        assert data["name"] == label
+        assert data["name"] == "Partial Dependence: " + label
```

Comment:
Ended up changing the PD trace name to mention that it is the partial dependence line to differentiate it from the ICE plots.